go.mod

```diff
@@ -1,32 +1,32 @@
 module github.com/snapcore/snapd
 
-go 1.18
+go 1.23.0
 
 // maze.io/x/crypto/afis imported by github.com/snapcore/secboot/tpm2
 replace maze.io/x/crypto => github.com/snapcore/maze.io-x-crypto v0.0.0-20190131090603-9b94c9afe066
 
 require (
-	github.com/bmatcuk/doublestar/v4 v4.6.1
+	github.com/bmatcuk/doublestar/v4 v4.9.1
```

Comment:
This needs to be updated in Debian: https://salsa.debian.org/go-team/packages/golang-github-bmatcuk-doublestar